### PR TITLE
Add initial auto-deploy for master branch

### PR DESCRIPTION
**Pull request by KrauseFx**
Auto deploy to https://docs.fastlane.tools

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ site_name: fastlane docs
 theme: readthedocs
 site_author: "fastlane team"
 site_description: "Documentation for fastlane tools, the easiest way to automate building and releasing your iOS and Android apps"
-strict: true
+# strict: true
```
Comment (mfurtak) [inline mkdocs.yml]: Intentional?

Comment (KrauseFx) [inline mkdocs.yml]: Yeah, for now, since deploy will make sure there are no deadlinks, and we currently do have dead links.

Comment (KrauseFx) [inline mkdocs.yml]: We should fix this, once we have the overall structure of the docs 👍

**Comment by mfurtak** [comment]
🎉

**Comment by mfurtak** [comment]
:ship: it

**Pull request merged**